homeassistant/components/binary_sensor/hive.py

```diff
@@ -35,9 +35,24 @@ def __init__(self, hivesession, hivedevice):
         self.attributes = {}
         self.data_updatesource = '{}.{}'.format(self.device_type,
                                                 self.node_id)
-
+        self.uniqueid = '{}-{}'.format(self.node_id, self.device_type)
         self.session.entities.append(self)
 
+    @property
+    def unique_id(self):
```

Comment:
Please move this addition to a separate PR.

Reply:
This code change has been moved to a new PR